### PR TITLE
Grading policy APP

### PR DESCRIPTION
**Pull request by polesye**
These methods were moved without changes:
- courseware.grades.py:
  - _grade
  - _progress_summary
- xmodule.course_module:
  - grading_context
- progress_summary part of the progress.html to grading_policy/templates/summary_seq.html

### CODE REVIEW

File: common/lib/xmodule/xmodule/course_module.py [hunk 1]
```diff
@@ -1288,6 +1294,16 @@ def _sorting_dates(self):
 
         return announcement, start, now
 
+
```
Comment (polesye) [inline common/lib/xmodule/xmodule/course_module.py]: remove empty line.

File: common/lib/xmodule/xmodule/course_module.py [hunk 2]
```diff
@@ -1289,6 +1295,15 @@ def _sorting_dates(self):
         return announcement, start, now
 
     @lazy
+    def grading(self):
```
Comment (auraz) [inline common/lib/xmodule/xmodule/course_module.py]: Please add comment why we go this way.

Comment (den99f) [inline common/lib/xmodule/xmodule/course_module.py]: > Please add comment why we go this way.

Please, ask @polesye for this information

Comment (polesye) [inline common/lib/xmodule/xmodule/course_module.py]: > Please, ask @polesye for this information

He asked me :)

Comment (polesye) [inline common/lib/xmodule/xmodule/course_module.py]: > Please add comment why we go this way.

I don't think that it should be a part of the docstring, we can add this info in PR to edx.

File: openedx/core/djangoapps/util/testing.py [hunk 3]
```diff
@@ -100,6 +101,7 @@ def setUp(self):
         self.course = self.store.get_item(self.course.location)
 
 
+@override_settings(GRADING_TYPE='sequential')
```
Comment (polesye) [inline openedx/core/djangoapps/util/testing.py]: remove this line.

File: openedx/core/djangoapps/util/testing.py [hunk 4]
```diff
@@ -3,6 +3,7 @@
 from datetime import datetime
 from pytz import UTC
 
+from django.test.utils import override_settings
```
Comment (polesye) [inline openedx/core/djangoapps/util/testing.py]: remove this line.

**Pull request closed without merging**